### PR TITLE
fix: embedding fixup for nomic-embed-text model

### PR DESCRIPTION
**Pull request by vansangpfiev**
For model like nomic-embed-text-v1.5.f16.gguf, we skip warm up.
Also change the API to send_embedding for non-logits model.

### CODE REVIEW

File: controllers/llamaCPP.cc [hunk 1]
```diff
@@ -649,6 +654,8 @@ bool llamaCPP::LoadModelImpl(std::shared_ptr<Json::Value> jsonBody) {
 
     params.n_gpu_layers = jsonBody->get("ngl", 100).asInt();
     params.n_ctx = jsonBody->get("ctx_len", 2048).asInt();
+    is_embedded_model =
```
Comment (vansangpfiev) [inline controllers/llamaCPP.cc:657]: use embedding flag to check if we should warm up model with embedding or not

**Review by tikikun — APPROVED** [review]
LGTM
